roles/install-bor/tasks/main.yml

```diff
@@ -1,26 +1,25 @@
 ---
 # tasks file for bor
 
-- name: Cloning the Bor
-  git:
-    repo: "https://github.com/maticnetwork/bor.git"
-    dest: "{{ ansible_env.HOME }}/bor"
-    force: yes
-    version: "{{ bor_branch }}"
+- name: install bor pacakges and bor profile
+  ansible.builtin.shell:
+    cmd: curl -L https://raw.githubusercontent.com/maticnetwork/install/main/bor.sh | bash -s -- {{ bor_version }} {{ network }} {{ node_type }}
 
-- name: Build Bor
-  command: chdir={{ ansible_env.HOME }}/bor make bor-all
+- name: uncomment p2p discovery
+  ansible.builtin.command:
+    cmd: sudo sed -i 's|.*\[p2p.discovery\]|  \[p2p.discovery\] |g' /var/lib/bor/config.toml
 
-- name: Creating symlink for bor
-  file:
-    src: "{{ ansible_env.HOME }}/bor/build/bin/bor"
-    dest: "/usr/bin/bor"
-    state: link
-  become: true
+- name: make bor the user of data directory
+  ansible.builtin.command:
+    cmd: sudo chown bor /var/lib/bor
```

Comment:
Eventually, it might be clearer to switch from `sudo` to using `become: true` but not critical

Reply:
fixed